replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorHistoryImpl.java

```diff
@@ -46,13 +62,40 @@
 
   public ReplicatorHistoryImpl(
       CatalogFramework framework,
+      CatalogProvider provider,
       FilterBuilder filterBuilder,
       MetacardHelper helper,
       MetacardType metacardType) {
+    this(framework, provider, filterBuilder, helper, metacardType, Security.getInstance());
+  }
+
+  public ReplicatorHistoryImpl(
```

Comment:
Only for testing?

Reply:
Added a comment that this is only for testing. Made it package private as well.